docs/usage_guide.rst

```diff
@@ -99,7 +99,7 @@ Bear in mind that in case of using multiple accounts/devices with the data sync
    :alt: Replace local database with remote backup
    :align: center
 
-You can manually move the global DB that contains the assets from one system to the other too. Find the :ref:`rotki_data_directory` in the source system. Assuming it's linux it will be ``~/.local/share/rotki/data``. The global db is then ``~/.local/share/rotki/data/global_data/global.db``. Manually move it to the equivalent location in the new system.
+You can manually move the global DB that contains the assets from one system to the other too. Find the :ref:`rotki_data_directory` in the source system. Assuming it's linux it will be :file:`~/.local/share/rotki/data`. The global db is then :file:`~/.local/share/rotki/data/global_data/global.db`. Manually move it to the equivalent location in the new system.
```

Comment:
or this file thing?

Reply:
The addition does not make a visual representation difference.
But it was from the [https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html?highlight=kbd#role-file](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html?highlight=kbd#role-file).